src/types/TestObject.test.ts

```diff
@@ -0,0 +1,212 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import * as Test from './TestObjects'
+
+describe('TestObject', () => {
+
+    const ALPHA = "alpha"
+    const BETA = "beta"
+    const CHARLIE = "charlie"
+    const conversationId = "1234" 
+
+    const makePair = (sourceA: string, sourceB: string, result: Test.RatingResult): Test.RatingPair => {
+        const sourceNames = [sourceA, sourceB].sort()
+        return {
+            conversationId,
+            sourceNames: [sourceNames[0], sourceNames[1]],
+            result
+        }
+    }
+
+    const addPair = (testSet: Test.TestSet, sourceA: string, sourceB: string, result: Test.RatingResult): void => {
```

Comment:
The function seems unnecessary.  It doesn't offer more value then the line itself.

Reply:
I think it makes the code easier to read